### PR TITLE
auto-complete not working in php 

### PR DESCRIPTION
**Pull request by aslijiasheng**
Thank you for contributing to Spacemacs!

Before you submit this pull request, please ensure it is against the develop branch and not master.

This message should be replaced with a description of your change.

Thank you <3

Im add feature-mode ac-php and configure in dotspacemacs-additional-packages add company-php, restart spacemacs and open phpfile, input php function-array_map anymore, show php tips;

for example https://gyazo.com/cf45a1d5108c20317dfd5ff9a885ea81

### CODE REVIEW

File: layers/+lang/php/packages.el [hunk 1]
```diff
@@ -24,24 +26,46 @@
         phpunit
         ))
 
-(when (configuration-layer/layer-usedp 'auto-completion)
-  (defun php/post-init-company ()
-    (spacemacs|add-company-hook php-mode)))
+(defun php/post-init-company ()
+  (spacemacs|add-company-hook php-mode))
+
+(add-hook 'php-mode-hook
```
Comment (d12frosted) [inline layers/+lang/php/packages.el]: This should not be at the top level.

Comment (aslijiasheng) [inline layers/+lang/php/packages.el]: @d12frosted  why?

**Comment by robbyoconnor** [comment]
This is against `master` and not `develop` -- Please close this and reopen it against `develop`.

**Comment by robbyoconnor** [comment]
To quote the message you ignored: 

> Before you submit this pull request, please ensure it is against the develop branch and not master.

**Comment by robbyoconnor** [comment]
This should be closed probably as it's invalid. @d12frosted @TheBB @syl20bnr

**Comment by robbyoconnor** [comment]
Also, before you make your PR -- switch to the `develop` branch so that you don't get massive conflicts.

**Comment by aslijiasheng** [comment]
@robbyoconnor  thank you , Im close it and reopen it

**Pull request closed without merging**